### PR TITLE
raftstore: correct the description on `inspect-interval` from v7.5

### PR DESCRIPTION
**Pull request by LykxSassinator**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Correct the descriptions of `raftstore.inspect-interval` in TiKV part, which has been changed from `500ms` to `100ms` by default from v7.5.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/tikv/tikv/pull/16087

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by LykxSassinator** [comment]
This pr needs to be cherry-picked to `v7.6`, `v8.0` and `v8.1`. /cc @qiancai

**Review by MyonKeminta — APPROVED** [review]

File: releases/release-7.5.0.md [hunk 1]
```diff
@@ -172,6 +172,7 @@ TiDB 7.5.0 为长期支持版本 (Long-Term Support Release, LTS)。
 | 配置文件 | 配置项 | 修改类型 | 描述 |
 | -------- | -------- | -------- | -------- |
 | TiDB | [`tikv-client.copr-req-timeout`](/tidb-configuration-file.md#copr-req-timeout-从-v750-版本开始引入) | 新增 | 设置单个 Coprocessor request 的超时时间。|
+| TiKV | [`raftstore.inspect-interval`](/tikv-configuration-file.md#inspect-interval) | 修改 | 经过算法调优后，调整慢节点检测的阈值以提升慢节点检测的灵敏度。从 v7.5.0 开始，默认值由 `500ms` 调整为 `100ms`。|
```
Comment (qiancai) [inline releases/release-7.5.0.md:175]:
```suggestion
| TiKV | [`raftstore.inspect-interval`](/tikv-configuration-file.md#inspect-interval) | 修改 | 经过算法调优后，默认值由 `500ms` 调整为 `100ms`，以提升慢节点检测的灵敏度。从 v7.5.0 开始，。|
```

File: releases/release-7.5.0.md [hunk 2]
```diff
@@ -172,6 +172,7 @@ TiDB 7.5.0 为长期支持版本 (Long-Term Support Release, LTS)。
 | 配置文件 | 配置项 | 修改类型 | 描述 |
 | -------- | -------- | -------- | -------- |
 | TiDB | [`tikv-client.copr-req-timeout`](/tidb-configuration-file.md#copr-req-timeout-从-v750-版本开始引入) | 新增 | 设置单个 Coprocessor request 的超时时间。|
+| TiKV | [`raftstore.inspect-interval`](/tikv-configuration-file.md#inspect-interval) | 修改 | 经过算法调优后，默认值由 `500ms` 调整为 `100ms`，以提升慢节点检测的灵敏度。从 v7.5.0 开始。|
```
Comment (qiancai) [inline releases/release-7.5.0.md:175]:
```suggestion
| TiKV | [`raftstore.inspect-interval`](/tikv-configuration-file.md#inspect-interval) | 修改 | 经过算法调优后，默认值由 `500ms` 调整为 `100ms`，以提升慢节点检测的灵敏度。|
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**